origins_workflow/tests/src/Nightwatch/Tests/AuthenticatedTest.js

```diff
@@ -7,30 +7,39 @@ module.exports = {
 
     browser
       .drupalRelativeURL('/node/add')
-      .expect.element('h1.page-title')
-      .text.to.contain('Access denied');
+      .expect.element('body')
+      .text.to.contain('You are not authorized to access this page');
 
     browser
-      .drupalRelativeURL('/admin/workflow/drafts')
-      .expect.element('h1.page-title')
-      .text.to.contain('Access denied');
+      .drupalRelativeURL('/admin/content')
+      .expect.element('body')
+      .text.to.contain('You are not authorized to access this page');
 
     browser
-      .drupalRelativeURL('/admin/workflow/all-drafts')
-      .expect.element('h1.page-title')
-      .text.to.contain('Access denied');
+      .drupalRelativeURL('/admin/content/media')
+      .expect.element('body')
+      .text.to.contain('You are not authorized to access this page');
```

Comment:
This makes a lot of sense, except when you combine with NIDirect 😂 

`/admin/content/media` gives the correct response code (403) but, due to a little bit of security theatre, it's been dressed up to look like a 404 response not a 403 with the intention of not advertising it as a system path and just pretending it doesn't exist instead.